README.rst

```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
 
    http://statsmodels.github.io/dev/index.html
```

Comment:
we should replace this, or add `www.statsmodels.org/dev/index.html`

"www" is required, doesn't work without